### PR TITLE
Bump release notes for RC2

### PR DESCRIPTION
**Pull request by andrewvc**

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -15,6 +16,15 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6.0.0-rc2]]
+=== Logstash 6.0.0-rc2 Release Notes
+
+* https://github.com/elastic/logstash/commit/840439722d8ef4737c7e8101c59652ced191bbea[Improved performance]
+* Fixed bug where logstash would crash when logging using the `debug` verbosity level
```
Comment (robbavey) [inline docs/static/releasenotes.asciidoc]: I'm not sure if this was an issue for RC1, just 5.6.3  - does it matter?

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -15,6 +16,15 @@ See also:
...
+* https://github.com/elastic/logstash/commit/546951fa889902d8ec56f8a7cec1dc41a21088ff[Modules: Set default credentials for Kibana if es info is present]
+* More small consistency fixes for Windows 
+* Fix bug where Elasticsearch output would not correctly log errors but rather an exception in the error handler
```
Comment (robbavey) [inline docs/static/releasenotes.asciidoc]: Feels like there is a word or two missing from this description

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -3,6 +3,7 @@
 
 This section summarizes the changes in the following releases:
 
+* <<logstash-6-0-0-rc1,Logstash 6.0.0-rc2>>
```
Comment (robbavey) [inline docs/static/releasenotes.asciidoc]: Link points to rc1, not rc2

File: docs/static/releasenotes.asciidoc (hunk 1 above)
Comment (andrewvc) [reply docs/static/releasenotes.asciidoc]: You are right! Will remove

Comment (robbavey) [reply docs/static/releasenotes.asciidoc]: Still present... ;)

**Pull request merged**